src/net/sourceforge/kolmafia/session/BastilleBattalionManager.java

```diff
@@ -3,14 +3,7 @@
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.PrintStream;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeMap;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
oops ;)

heeheehee [posted an xml file for formatting](https://kolmafia.us/threads/mafia-source-code-formatting.26013/#post-161996) a while ago. It might do with adding to the repo?

Reply:
Irritating. It has a tool to "fix" those - on commit. I will fix all of this.

Yeah. We should have a standard file of code style, starting with heeheehee's
spotless will have to agree with it.
I also think spotless makes some things really ugly and hard to read.

Reply:
I fixed this.

I grepped and found another 15 or so packages that have "*" imports for Java utilities, KoLmafia utilities, KoLmafia persistence, KoLmafia requests ...

I did not touch those in the PR - although I would have, had IntelliJ's automated tool to fix them been callable from the Inspections frame.

Reply:
I tried to move some discussion to https://kolmafia.us/threads/source-code-formatting.28429/

I'm not sure the xml file for IntelliJ is still useful given the inclusion of spotless in the workflow but if spotless is not enforcing what we want we should try and "fix" spotless.

Reply:
> I also think spotless makes some things really ugly and hard to read.

Definitely agreed for some things: I think the way it indents comments can be really dumb, and the way it spreads things across lines can also make it harder to read.

We could see if we could configure it to make it more amenable. I'm not sure there are any alternatives in this space.